document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -55,6 +56,6 @@ public DocumentInfo getDocumentInfo() {
      * @return true if abridged, false if not
      */
     private boolean isAbridged(String accountLink) {
-        return accountLink.matches("/transactions/\\d+(\\-\\d+)+/accounts/\\w+?=");
+        return accountLink.matches("/transactions\\/[0-9\\-]+\\/accounts\\/.*");
```

Comment:
You shouldn't need the escape of the `-` inside the block. Intellij tends to complain about this anyway.

Reply:
fixed